_data/team-management.yml

```diff
@@ -80,3 +80,17 @@
     - Management
     - Finance
 
+- name: Bob Carroll
+  title: CTO
+  description:
+  image: anon.png
+  email:
+  keybase:
+  image: bob_carroll.jpg
+  email: bob@dash.org
+  keybase: https://keybase.io/bobcarroll
+  twitter:
+  dashforum-link: /bob.27367/
+  dashforum: bob
+  team-title:
+    - Management
```

Comment:
Can you also add "Development" team here?

Reply:
@nmarley On the next line